pkg/archive/changes_test.go

```diff
@@ -42,16 +41,6 @@ type FileData struct {
 	permissions os.FileMode
 }
 
-// resetSymlinkTimesLinux sets the atime and mtime of a symlink to a known value, to test
-// whether changes to the symlink target are detected correctly.
-func resetSymlinkTimesLinux(path string) error {
-	if runtime.GOOS != linux {
```

Comment:
nit: with that, `const linux` in archive.go (introduced by commit f5c1ad912) is now unused.